### ISSUE TITLE
Make ubuntu-core a team of ubuntu

### ISSUE THREAD

**Issue by elopio** [ubuntu-core/snappy#4]
There is already an ubuntu organization in github: https://github.com/ubuntu

I think it would be better to keep all ubuntu repositories in that organization. And we can make a team in there called ubuntu-core or snappy. @didrocks is probably an owner of that org, or knows who the owner is.

**Comment by didrocks** [comment]
Yeah, just ping me once you created your team, and I'll add you to the ubuntu organization.

**Comment by niemeyer** [comment]
I'd prefer to keep it here for now, if that's okay, so projects that are related to ubuntu-core and snappy are all together and follow common conventions. There isn't much of a down side, I think.

**Comment by elopio** [comment]
If we don't do it now, later we would have to replace all the go import paths again. The ubuntu org currently has only two projects, so it won't be hard to follow common conventions there. And later when more projects are added, it will be easier to tell them to follow the same conventions.

And maybe, by putting all the ubuntu projects together we encourage people to contribute on all of them. Or make it easier to find a project they like.

So I'd prefer to use the existing "Ubuntu" org + a team now. But if you feel strongly about keeping an ubuntu-core org, I can live with that too.

**Comment by niemeyer** [comment]
It's unreasonable to arbitrate on all projects under "ubuntu" in terms of conventions. The scope is extremely wide, with diverse teams, communities, languages, etc. Also, replacing Go import paths is not a major problem. If we decide to do that later, we can. But let's please start small and focused.

You said you were okay with that if I felt this was the right thing to do for now, so I'll take your offer and close this issue for the time being.
